ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -19,12 +19,14 @@ import {ZeroState} from '../../../shared/components/zero-state';
 import {exampleWorkflow} from '../../../shared/examples';
 import {Utils} from '../../../shared/utils';
 
-import {WorkflowFilters} from '../workflow-filters/workflow-filters';
+import {FilterType, WorkflowFilters} from '../workflow-filters/workflow-filters';
 
 require('./workflows-list.scss');
 
 interface State {
     loading: boolean;
+    checkedServiceInfo: boolean;
+    isManagedNamespace: boolean;
```

Comment:
I think this should just be `managedNamespace`

Reply:
done.